client/src/app/site/pages/organization/pages/committees/pages/committee-detail/modules/committee-detail-edit/components/committee-detail-edit/committee-detail-edit.component.ts

```diff
@@ -89,7 +89,11 @@ export class CommitteeDetailEditComponent extends BaseComponent implements OnIni
      * @returns A function that will return a boolean
      */
     public getDisableOptionWhenFn(): (value: Selectable) => boolean {
-        return value => value.id === this.committeeId;
+        return value => value.id === this.committeeId || !(this.isOrgaManager || this.committeeRepo.getViewModel(value.id).manager_ids?.includes(this.operator.user.id));
```

Comment:
If committee management status is checked like this, committee managers will not be able to add subcommittees of their own committee if they're not immediately admin there.

Perhaps consider using `this.operator.hasCommitteePermissions` for everything in the round brackets or build a new operator.service function
```
public committeeCanManageNoOrgaCheck(committeeId: number): boolean {
   return this._CML[committeeId] === CML.can_manage;
}
```
and use that instead of the last committeeRepo check